routes/api/v1/foods.js

```diff
@@ -77,4 +77,25 @@ router.delete("/:id", function (req, res, next) {
   })
 });
 
+/*UPDATE a food given the id*/
+router.patch("/:id", function (req, res, next) {
+  return Food.update({
+    name: req.body.name,
```

Comment:
Should we be using Joi here too?

Reply:
Let's add a test for only updating one attribute to make sure it acts as expected

Reply:
Added test on branch